stdlib/package.py

```diff
@@ -137,6 +139,176 @@ def __init__(
+
+    def drain_package(self, source, *paths: str):
+        """Drain a :py:class:`.Package`, moving files from its ``wrap_cache`` to this :py:class:`.Package`'s ``wrap_cache``.
+
+        This function is mostly used to grab the files that were wrongly assigned to another package by the package splitter.
+
+        *Example:* ::
+
+            lib_pkg.drain_package(
+                bin_pkg,
+                'usr/lib{32,64}/',
+                'usr/lib/lib*.so',
+            )
+
+        :note: Files are moved and not copied
+
+        :param source: The source package containg the files to drain from.
+        :type source: :py:class:`.Package`
+        :param paths: The paths pointing to the files to move, relative to the ``wrap_cache`` of the given :py:class:`.Package`.
+            This argument supports the globbing and braces syntax of common shells.
+        """
+        with stdlib.pushd(source.wrap_cache):
+            for rglob in paths:
+                for rglob in braceexpand.braceexpand(rglob):  # Expand braces
+
+                    if os.path.isabs(rglob):
+                        raise RuntimeError("Package.drain_package() received an absolute path as parameter, but it expects a relative one")
+
+                    for rpath in glob.glob(rglob):  # Expand globbing
+                        dstpath = os.path.join(
+                            self.wrap_cache,
+                            os.path.relpath(
+                                rpath,
+                                source.wrap_cache,
+                            ),
+                        )
+
+                        os.makedirs(os.path.dirname(dstpath), exist_ok=True)  # Create parent directories (if any)
+                        shutil.move(rpath, dstpath)
+
+    def drain_build_cache(self, src, dst):
+        """Drain the current :py:class:`.Build`, moving files from its ``build_cache`` to this package's ``wrap_cache``.
+
+        This function is mostly used to grab files that are given with the software's source code but aren't
+        automatically installed, like the documentation or some code examples.
+
+        *Example*: ::
+
+            package.drain_build_cache(
+                'doc/*',
+                'usr/share/doc/my_package_0.1.1/'
+            )
+
+        :note: Files are moved and not copied
+
+        :param src: A path pointing to the files to move, relative to the ``build_cache`` of the current :py:class:`.Build`,
+            This argument supports the globbing and braces syntax of common shells.
+        :param dst: A path pointing to the directory where the files should be moved, relative to the ``wrap_cache`` of this :py:class:`.Package`,
+
+        """
+        build = stdlib.build.current_build()
+
+        if os.path.isabs(src):
+            raise RuntimeError("Package.drain_build_cache() received an absolute path as parameter, but it expects a relative one")
+        if os.path.isabs(dst):
```

Comment:
This condition could be also put in the same `if` as the one just above, although it's not that big of a deal. The same thing happens in the next function

Reply:
Yep you're right!